src/platform/packages/private/kbn-grid-layout/grid/use_grid_layout_events/section/events.ts

```diff
@@ -83,18 +85,17 @@ export const useGridLayoutRowEvents = ({ rowId }: { rowId: string }) => {
           onEnd,
         });
       } else if (isKeyboardEvent(e)) {
-        const isEventActive = gridLayoutStateManager.activeRowEvent$.value !== undefined;
+        if (gridLayoutStateManager.activeSectionEvent$.getValue()) return; // interaction has already happened, so don't start again
         startKeyboardInteraction({
           e,
-          isEventActive,
           onStart,
           onMove,
           onEnd,
           onCancel,
         });
       }
     },
-    [gridLayoutStateManager, rowId, onEnd, onCancel]
+    [gridLayoutStateManager, sectionId, onEnd, onCancel]
   );
 
   return { startDrag: startInteraction, onBlur: onEnd };
```

Comment:
startKeyboardInteraction({
          e,
          onStart,
          onMove,
          onEnd,
          onBlur: onEnd,
          onCancel,
        });
      }
    },
    [gridLayoutStateManager, sectionId, onEnd, onCancel]
  );

  return { startDrag: startInteraction };

Reply:
We still need the `onBlur`, no?

Reply:
no, I actually suggest removing onBlur from here but add it internally here:
```
startKeyboardInteraction({
          e,
          onStart,
          onMove,
          onEnd,
          onBlur: onEnd, // here 
          onCancel,
        });
```

This makes more sense with how we try to stucture things here because we don't expose the event flow directly to the drag/resize handles, just attach the event.

Reply:
In [this comment](https://github.com/elastic/kibana/pull/208286#discussion_r2016527138), you said it was necessary to have the `onBlur` event on the section header when I asked about it - I didn't actually change this code. Is this no longer the case?

Reply:
Ahh, that makes sense!! Thanks for explaining 🤦 I can clean this up!!

Reply:
Cleaned up in a57f8d2e6665ad85992646ffbc71ceaab2325635